src/fosslight_source/cli.py

```diff
@@ -91,6 +91,9 @@ def main():
 
     _start_time = datetime.now().strftime('%y%m%d_%H%M')
     success, msg, output_path, output_file, output_extension = check_output_format(output_file_name, format)
+    if output_extension in ['.csv', '.json', '.yaml']:
```

Comment:
`if output_extension in ['.csv', '.json', '.yaml']:` ->` if output_extension in ['.csv', '.json', '.yaml']  and print_matched_text:`

Reply:
Done :)